pages/box-office/_performanceId/sell/complete.vue

```diff
@@ -34,41 +60,74 @@ export default {
       type: Booking,
     },
   },
-  async mounted() {
+  data() {
+    return {
+      checkedIn: false,
+      currentTime: new Date(),
+    }
+  },
+  mounted() {
     if (!this.booking.reference) return this.$router.push('../')
 
-    try {
-      await performMutation(
-        this.$apollo,
-        {
-          mutation: CheckInTickets,
-          variables: {
-            reference: this.booking.reference,
-            performanceId: this.booking.performance.id,
-            tickets: this.booking.tickets.map((ticket) => {
-              return {
-                ticketId: ticket.id,
-              }
-            }),
-          },
-        },
-        'checkInBooking'
-      )
-      successToast.fire({
-        timer: 4000,
-        title: 'Tickets automatically checked in',
-      })
-    } catch (e) {
-      errorToast.fire({
-        title: 'Unable to check in tickets automatically',
-      })
-    }
+    if (this.canAutoCheckIn) this.changeTicketStatus(true)
```

Comment:
98% sure that this needs to be `this.canAutoCheckIn()` (i.e. with brackets) to execute the function. Same on line 80